cumulusci/core/flowrunner.py

```diff
@@ -374,16 +392,17 @@ def _visit_step(
             step_overrides.update(step_config.get("options", {}))
             task_config["options"].update(step_overrides)
 
+            # merge runtime options
+            if name in self.runtime_options:
+                task_config["options"].update(self.runtime_options[name])
```

Comment:
as i mentioned in a chat at one point, these are all updates instead of dictmerges which is what we do at the yaml level. should we switch?

Reply:
I think update is both more backwards-compatible and less surprising. We can re-evaluate if a specific use case presents itself.